src/citrine/resources/dataset.py

```diff
@@ -349,3 +349,30 @@ def register(self, model: Dataset) -> Dataset:
         full_model = self.build(data)
         full_model.project_id = self.project_id
         return full_model
+
+    def list(self,
+             page: Optional[int] = None,
+             per_page: int = 1000) -> Iterable[Dataset]:
```

Comment:
This is an overriden method?

And we're changing the default per_page to 1000 for everyone? (not just in the tests)?

Reply:
Yes it's an overridden method and this default applies to everyone using citrine-python.

Applying this to the e2es only would require the sort of solution I mentioned to you earlier, e.g. a keyword arg for find_collection() that would let us pass this value to the various collections.